Source/SIMPLib/Filtering/FilterPipeline.cpp

```diff
@@ -869,34 +891,33 @@ DataContainerArray::Pointer FilterPipeline::execute(DataContainerArray::Pointer
 
     // Emit that the filter is completed for those objects that care, even the disabled ones.
     emit filt->filterCompleted(filt.get());
+
+    notifyProgressMessage(static_cast<int>(static_cast<float>(filtIndex + 1) / (m_Pipeline.size()) * 100.0f), "");
   }
 
   disconnectSignalsSlots();
 
-  PipelineMessage completeMessage("", "", 0, PipelineMessage::MessageType::StatusMessage, -1);
-  if(m_State == FilterPipeline::State::Canceling)
-  {
-    completeMessage.setText("Pipeline Canceled");
-    m_ExecutionResult = FilterPipeline::ExecutionResult::Canceled;
-  }
-  else if(m_State == FilterPipeline::State::Executing)
+  switch(m_State)
```

Comment:
Check your switch state formatting.  Cases should be at the same indentation as the switch.

Reply:
You also don't need curly brackets unless you create a variable inside a case block.